src/sentry/models/integrations/external_issue.py

```diff
@@ -55,7 +55,7 @@ class ExternalIssue(Model):
     # The foreign key here is an `int`, not `bigint`.
     integration = FlexibleForeignKey("sentry.Integration", db_constraint=False)
 
-    key = models.CharField(max_length=128)  # example APP-123 in jira
```

Comment:
Oof I might have misunderstood, this means that it would probably show up as wayyyy long in the UI with a linked issue.

I thought it was the hyper-link but that might not be the case.

Reply:
Yeah that's what I was a little worried about, but it's only happening to 2 organizations and I think if I were them I'd prefer a long wonky link to none at all 🤷‍♀️

Reply:
Not that bad, we're okay. Plus 128 was already triggering the ellipsis. LGTM

<img width="324" alt="image" src="https://user-images.githubusercontent.com/35509934/161860955-50c97d9e-dd03-4b72-90f9-df1c7a35f88b.png">

Reply:
The icon in that screenshot looks smooshed. It might need to have `flex-shrink: 0` applied to it.

Reply:
Fixed that here: https://github.com/getsentry/sentry/pull/33357